index.html

```diff
@@ -302,26 +303,27 @@
       <h2>Introduction</h2>
 
       <p>
-[=Credentials=] are a part of our daily lives; driver's licenses are used to
-assert that we are capable of operating a motor vehicle, university degrees
-can be used to assert our level of education, and government-issued passports
-enable us to travel between countries. These [=credentials=] provide
+[=Credentials=] are integral to our daily lives; driver's licenses confirm
+our capability to operate motor vehicles, university degrees assert our level
+of education, and government-issued passports permit travel between countries.
+This specification provides a mechanism to express these sorts of
+[=credentials=] on the Web in cryptographically secure, privacy-respecting,
+and machine-verifiable way. These [=credentials=] provide
```

Comment:
This paragraph fully repeats the `Abstract`. That *reads* as an error, even if it is intentional (which I do not think it should be).
```suggestion
[=Credentials=] are integral to our daily lives: driver's licenses confirm
our capability to operate motor vehicles, university degrees assert our level
of education, and government-issued passports attest to our citizenship when traveling between countries.
This specification provides a mechanism to express these sorts of
[=credentials=] on the Web in a cryptographically secure, privacy-respecting,
and machine-verifiable way. These [=credentials=] provide
```